tests/virt_autotest/sriov_network_card_pci_passthrough.pm

```diff
@@ -238,25 +245,24 @@ sub prepare_guest_for_sriov_passthrough {
                 assert_script_run "sed -i '/<currentMemory/d' $vm.xml";
                 record_info "Disable guest ballooning", "$vm";
             }
-            #enable pci-passthrough and set e820_host for pv guest
-            #enable pci-passthrough for fv guest on sles15sp2+
-            #refer to bug #1167217 for the reason
-            #but need wait libvirt support for pv guest requested in a bug on sles12sp5
+            #enable pci-passthrough on sles15sp2+
+            #set e820_host for pv guest
+            #refer to bug #1167217 and but #1185081 for the reason
             if (is_pv_guest($vm) || (is_fv_guest($vm) && is_sle('>=15-SP2'))) {
-                unless (script_run("grep '<features>' $vm.xml") == 0) {
+                unless (script_run("xmlstarlet sel -t -c /domain/features $vm.xml") == 0) {
                     assert_script_run "xmlstarlet edit -L -s /domain -t elem -n features -v '' $vm.xml";
                 }
+                unless (script_run("xmlstarlet sel -t -c /domain/features/xen $vm.xml") == 0) {
+                    assert_script_run "xmlstarlet edit -L -s /domain/features -t elem -n xen -v '' $vm.xml";
+                }
```

Comment:
assert_script_run "xmlstarlet edit -L -s /domain/features -t elem -n xen -v '' $vm.xml" unless (script_run("xmlstarlet sel -t -c /domain/features/xen $vm.xml") == 0); ?

Reply:
too long, just keep it as it was.